data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,160 @@ mission "Arachi Cartoonist 2"
+
+mission "Coalition: Submarines 2"
+	name "Pressurized Stress Test"
+	description "Bring Apri'ie and the <cargo> to <destination>, where the submarines will be subjected to some final high-pressure stress tests."
+	passengers 1
+	cargo "unmanned submarines" 60
+	blocked "You need <capacity> to take the next mission. Return here when you have the required space free."
+	to offer
+		has "Coalition: Submarines 1: done"
+	source "Market of Gupta"
+	destination "Inmost Blue"
+	on offer
+		conversation
+			`After asking around for a member of House Mallob, you're informed that Apri'ie hasn't left his meeting yet, and so you take a seat in the spaceport, occasionally getting up to check back at the entrance of the office he entered. After some hours, you finally see him coming out, accompanied by a Saryd speaking on a communication device as they glance at a data pad.`
+			`	"Captain <last>! Glad that you are interested in assisting us further, I am. Ready to have the submarines loaded into your cargo hold, are you?" Apri'ie asks. "Ready to conduct some final stress tests, colleagues of mine on <planet> are."`
+			choice
+				`	"Sure thing, I've set aside the space for them."`
+					goto end
+				`	"What sort of testing are you going to do?"`
+			`	"Capable of enduring rapid shifts in pressure, the subs must be, so take quick, deep dives in the oceans of <planet>, and then resurface just as quickly, they will," he says. "Performed their own tests, House Mallob has, so most assuredly ready to go, the vessels are, but just as a formality, perform these tests, we must."`
+			choice
+				`	"I see. Well, I'm ready to go, so you can have the submarines brought to my ship."`
+					goto end
+				`	"Why not just do those tests on Second Cerulean right away?"`
+			`	"Upsetting or startling the local fauna, we wish to avoid, so wait until after the tests, we will, before deploying them in that planet's waters."`
+			label end
+			`	The Saryd issues a command on the data pad and turns off the communication device. They then say that the cargo is being brought to your ship, and accompanies you and Apri'ie there. There are five submarines in total, strapped to large, sturdy sleds, which are slowly pushed up the ramp to your cargo hold. The subs themselves are somewhat oval in shape, and flat on the top and bottom. Other than the propulsion mechanisms on the back, you notice multiple dark dots protruding on the front, likely a system of cameras, and two thin "arms" a bit to the sides, retracted for easier storage of the crafts. After the last sub is safely in your cargo hold, Apri'ie heads into your ship, and you set a route for <destination>.`
+				accept
```

Comment:
I'm rather unfamiliar with ES mission syntax, but I think this has too many indents. 
```suggestion
			accept
```

Reply:
No, `accept` is indented one further than the conversation text that accepts the mission.

Reply:
Sorry about that.

Reply:
It's alright.